saltboot-formula/_states/saltboot.py

```diff
@@ -1392,7 +1402,7 @@ def image_deployed(name, partitioning, images):
             if 'hash' in image:
                 _write_image_version(device, '{0}-{1}'.format(image['name'], image_version), image['hash'], luks_pass)
 
-            if not __opts__['test'] and devmap[name].get('mountpoint') == '/':
+            if devmap[name].get('mountpoint') == '/':
```

Comment:
Why this?

Reply:
This I just noticed that is unneeded as the check for test is already in if on line 1344 and this is the else branch of that check.

Reply:
you are right